java/java.lsp.server/vscode/src/extension.ts

```diff
@@ -156,6 +157,13 @@ export function activate(context: ExtensionContext): VSNetBeansAPI {
 
     // find acceptable JDK and launch the Java part
     findJDK((specifiedJDK) => {
+        context.subscriptions.push(vscode.extensions.onDidChange(() => {
+            const currentClustersSorted = currentClusters.slice().sort();
```

Comment:
keep the global state somewhere here, if possible.

Reply:
Fixed.